src/types/opencv/Feature2D.d.ts

```diff
@@ -1,7 +1,7 @@
 import type { Algorithm, KeyPointVector, Mat, OutputArray } from "./_types";
 
 /**
- * https://docs.opencv.org/master/d0/d13/classcv_1_1Feature2D.html
+ * https://docs.opencv.org/4.12.0/d0/d13/classcv_1_1Feature2D.html
```

Comment:
Better to use a fixed version of the link. The link with `master` might get broken when a new major version is released.